tests/test_sourcemap.py

```diff
@@ -1,4 +1,4 @@
-from typing import Iterator
+from collections.abc import Iterator
```

Comment:
what's the story behind this change? I saw this somewhere else and didn't get why

Reply:
typing has aliases to collections.abc but theyre deprecated. Pre python 3.9, they were required